lib/puppet/pops/parser/lexer2.rb

```diff
@@ -535,7 +535,6 @@ def initialize()
             scn.pos = before
             invalid_number = scn.peek(after - before) unless invalid_number
           end
-          length = scn.pos - before
```

Comment:
@hlindberg @thallgren I believe this was copied/pasted from elsewhere, but not needed

Reply:
Not needed - it is not using `length` since it is on the "error branch" of the logic and does not produce a token.